backend-beeyond/src/main/java/at/htl/beeyond/entity/User.java

```diff
@@ -14,7 +14,7 @@
 
     private String name;
 
-    @OneToMany(mappedBy = "user")
+    @ManyToMany
```

Comment:
If more than 3 tables are generated because of this `@ManyToMany`, you may need a `mappedBy`

Reply:
Das mappedby ist bei der Entity Namespace